projects/plugins/jetpack/class.json-api.php

```diff
@@ -1021,26 +1021,37 @@ public function wp_count_comments( $post_id ) {
 		);
 
 		/**
-		 * Exclude certain comment types from comment counts in the REST API.
-		 *
-		 * @since 6.9.0
-		 * @module json-api
-		 *
-		 * @param array Array of comment types to exclude (default: 'order_note', 'webhook_delivery', 'review', 'action_log')
-		 */
-		$exclude = apply_filters(
-			'jetpack_api_exclude_comment_types_count',
-			array( 'order_note', 'webhook_delivery', 'review', 'action_log' )
+		* Exclude certain comment types from comment counts in the REST API.
+		*
+		* @since 6.9.0
+		* @deprecated $$next-version$$
+		* @module json-api
+		*
+		* @param array Array of comment types to exclude (default: 'order_note', 'webhook_delivery', 'review', 'action_log')
+		*/
+		$exclude = apply_filters_deprecated( 'jetpack_api_exclude_comment_types_count', array( 'order_note', 'webhook_delivery', 'review', 'action_log' ), 'jetpack-11.0', 'jetpack_api_include_comment_types_count' ); // phpcs:ignore VariableAnalysis.CodeAnalysis.VariableAnalysis.UnusedVariable
```

Comment:
```suggestion
		$exclude = apply_filters_deprecated( 'jetpack_api_exclude_comment_types_count', array( 'order_note', 'webhook_delivery', 'review', 'action_log' ), 'jetpack-$$next-version$$', 'jetpack_api_include_comment_types_count' ); // phpcs:ignore VariableAnalysis.CodeAnalysis.VariableAnalysis.UnusedVariable
```

Reply:
@anomiex Should the `replace-next-version-tag` script be catching this already, or would it need to be adjusted?

```
[plugins/jetpack 4:17.704] === Updating $$next-version$$ ===
[plugins/jetpack 4:17.704] 
[plugins/jetpack 4:19.232] Processing projects/plugins/jetpack/class.json-api.php
[plugins/jetpack 4:19.243] ::error file=projects/plugins/jetpack/class.json-api.php,line=1032::Unexpected `$$next-version$$` token.
```

Reply:
It probably should, but currently doesn't. It only handles `_deprecated_function`, `_deprecated_constructor`, `_deprecated_file`, `_deprecated_argument`, and `_deprecated_hook`.

Handling `apply_filters_deprecated` may not be entirely straightforward, since it seems likely enough that phpcs might wind up wrapping that onto multiple lines. Maybe it's time to just replace `$$next-version$$` everywhere and hope that never manages to collide with anything unexpected.